app/cms/utils/format-links.ts

```diff
@@ -36,6 +35,9 @@ const formatLinks = (...args) => {
 
           href = href.replace(/(?<=[^/])#/, "/#")
           href = href.replace(".mdx", "").replace(".md", "")
+          // remove parent or current directory paths from href, ie. remove: `../` or `./`
```

Comment:
@neilZon @srinjoyc I'm assuming there are no pages more than 1 level deep, or no repos that contain multiple docs folders linking to each other?

Reply:
We should account for pages at an arbitrary depth. This will require either some redundancy or clever coding.

Reply:
This comment might be a bit misleading but it should say `removes all occurrences of the pattern '../' or './'`. But now that you mention it, this solution assumes that the lowest common ancestor is the docs folder, but it could be that the lowest common ancestor is lower in the directory tree when 1 file is giving a relative path to another so this will require a bit more thought

Reply:
Actually, we'll leave `docs` as the lowest common ancestor. This works!